compiler/cpp/src/generate/t_py_generator.cc

```diff
@@ -21,12 +21,19 @@
 #include <fstream>
 #include <iostream>
 #include <vector>
+#include <list>
+#include <map>
 
 #include <stdlib.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <sstream>
 #include <algorithm>
+#include <boost/graph/adjacency_list.hpp>
+#include <boost/graph/topological_sort.hpp>
+#include <boost/graph/depth_first_search.hpp>
+#include <boost/graph/visitors.hpp>
```

Comment:
We don't want to add boost as a dependency when building the Thrift compiler. Sorry.